src/app/core/entity-components/entity-list/filter-generator.service.ts

```diff
@@ -98,7 +114,7 @@ export class FilterGeneratorService {
     filter: BooleanFilterConfig
   ): FilterSelectionOption<T>[] {
     return [
-      { key: "all", label: filter.all, filterFun: () => true },
+      { key: "", label: filter.all, filterFun: () => true },
```

Comment:
This creates a problem with the configurable enums where we sometimes have the `NONE` option with a empty string as key.